onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -2277,12 +2283,12 @@ TEST(ReductionOpTest, ReduceDimWithZero) {
   };
 
   // reduce on all axes keeping dims. should allow the 0 to be the reduced value
-  OpTester test("ReduceSum", 10);
-  test.AddAttribute("keepdims", int64_t(1));
-  test.AddShapeToTensorData(true, 1);  // make second dim symbolic so that we don't break during shape inferencing
-  test.AddInput<float>("data", {3, 0, 2}, {});
-  test.AddOutput<float>("reduced", {1, 0, 1}, {});
-  run(test);
+  //OpTester test("ReduceSum", 10);
```

Comment:
What's the plan for re-enabling these tests?

Reply:
As discussed, enabled the test.